src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/InitialConsiderationService.java

```diff
@@ -44,6 +44,11 @@ public class InitialConsiderationService {
             + "|-------------|:------------|\r\n"
             + "|In ET1 by claimant | %s|\r\n"
             + "|In ET3 by respondent | %s|\r\n"
+            + "\r\n"
+            + "| Respondent hearing panel preference | |\r\n"
+            + "|-------------|:------------|\r\n"
+            + "|Preference | %s|\r\n"
+            + "|Reason | %s|\r\n"
```

Comment:
The ticket has this section being displayed under the hearing details section as it's own table as opposed to being under the respondent details. Despite this, I do think it makes sense that it is part of the respondent table but is it worth removing the new table heading and then changing 'Preference' to 'Hearing Panel Preference'.

Maybe one to double check with Dan.